src/components/organisms/SectionContents/gallery.jsx

```diff
@@ -3,6 +3,12 @@ import * as React from 'react'
 //** Additional Frameworks */
 import InstagramEmbed from 'react-instagram-embed';
 
+//** Helpers */
+//** Personalization */
+import Text from "../../helper/Text";
+import { renderToString } from 'react-dom/server';
+import ReactHtmlParser from 'react-html-parser'; 
+
 /**
  * Section: Instagram
  */
```

Comment:
This is disgusting drug-induced crap, and should die
To put it with Linus words.